sdks/java/extensions/sql/src/test/java/org/apache/beam/sdk/extensions/sql/BeamSqlDslUdfUdafTest.java

```diff
@@ -101,6 +103,21 @@ public void testJodaTimeUdfUdaf() throws Exception {
     pipeline.run().waitUntilFinish();
   }
 
+  @Test
+  public void testListUdf() throws Exception {
+    Schema resultType = Schema.builder().addArrayField("array_field", FieldType.INT64).build();
+
+    Row row1 = Row.withSchema(resultType).addValue(Arrays.asList(1L)).build();
```

Comment:
Nit: why are row1, sql1, etc. variable names numbered when there's only one of each?

Reply:
Done